aten/src/ATen/MemoryFormat.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
```

Comment:
we don't need to do this -- we have aten versions of the the things in c10 as deprecated usages from when these things were in ATen, so we don't break people's builds.  But if you have a new thing it doesn't matter.

Reply:
Do you mean I need to replace #include <ATen/MemoryFormat.h> with #include <c10/core/MemoryFormat.h> everywhere?

Reply:
I'm not really sure what needs to be in `c10/core` vs `c10`, but otherwise, yes.